frontend/packages/dev-console/src/components/formik-fields/PipelineResourceDropdownField.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react';
```

Comment:
I think this file should not be inside formik-field folder since it's not a generic field component and has knowledge about the form status like `subFormsOpened `. Better keep it in the pipelines folder along with related form components